cpp/models/abm/person.cpp

```diff
@@ -18,6 +18,7 @@
 * limitations under the License.
 */
 #include "abm/person.h"
+#include "abm/state.h"
 #include "abm/world.h"
 #include "abm/location.h"
 #include "memilio/utils/random_number_generator.h"
```

Comment:
@xsaschako Could yo have a look what happend with `person.h`? Github says it is an empty file with no changes...